Loop/Views/SettingsView.swift

```diff
@@ -216,18 +218,38 @@ extension SettingsView {
     }
     
     private var loopSection: some View {
-        Section(header: SectionHeader(label: localizedAppNameAndVersion)) {
-            Toggle(isOn: closedLoopToggleState) {
-                VStack(alignment: .leading) {
-                    Text("Closed Loop", comment: "The title text for the looping enabled switch cell")
-                        .padding(.vertical, 3)
-                    if !viewModel.isOnboardingComplete {
-                        DescriptiveText(label: NSLocalizedString("Closed Loop requires Setup to be Complete", comment: "The description text for the looping enabled switch cell when onboarding is not complete"))
-                    } else if let closedLoopDescriptiveText = viewModel.closedLoopDescriptiveText {
-                        DescriptiveText(label: closedLoopDescriptiveText)
+        Section(
+            header: SectionHeader(
+                label: NSLocalizedString("Tidepool Loop", comment: "Loop section header label")
+            )
```

Comment:
this should use either the `localizedAppNameAndVersion` provided or the `appName` environment variable instead of forcing Tidepool Loop to also support DIY Loop.

Reply:
@nhamming totally agree, so I made the change, however, design may kick this back as this will not match the design:

![Simulator Screenshot - iPhone 15 Pro - 2024-05-24 at 20 58 12](https://github.com/tidepool-org/Loop/assets/133469245/922e6527-dbb9-48a1-a45e-282f9bcbb979)